### PR TITLE
feat: get all rooms paginated

### LINKED ISSUE #148: Feat: get all rooms paginated

### PR DESCRIPTION
**Pull request by ChienNQuang**

### CODE REVIEW

File: src/Application/Rooms/Queries/GetAllRoomsPaginated.cs [hunk 1]
```diff
@@ -8,9 +13,50 @@ public class GetAllRoomsPaginated
...
+        }
+
+        public async Task<PaginatedList<RoomDto>> Handle(Query request, CancellationToken cancellationToken)
+        {
+            var rooms = _context.Rooms.AsQueryable();
+
+            if (!(request.SearchTerm is null || request.SearchTerm.Trim().Equals(string.Empty)))
+            {
+                rooms = rooms.Where(x =>
+                    x.Name.Contains(request.SearchTerm, StringComparison.InvariantCultureIgnoreCase));
```
Comment (kaitoz11) [inline src/Application/Rooms/Queries/GetAllRoomsPaginated.cs]: Will this work?

**Review by kaitoz11 — COMMENTED** [review]
Tell me if you done....?

File: src/Application/Rooms/Queries/GetAllRoomsPaginated.cs (hunk 1 above)
Comment (ChienNQuang) [reply src/Application/Rooms/Queries/GetAllRoomsPaginated.cs]: no

**Review by kaitoz11 — APPROVED** [review]
Test-less do more is our way of TDD

**Pull request merged**